### PR TITLE
Feature/adding jeeves

### PR DESCRIPTION
**Pull request by ahuston-0**

### CODE REVIEW

File: systems/configuration.nix [hunk 1]
```diff
@@ -75,7 +75,7 @@
         MaxSessions = lib.mkDefault 2;
         PasswordAuthentication = false;
         PermitEmptyPasswords = "no";
-        PermitRootLogin = "no";
+        PermitRootLogin = lib.mkDefault "no";
```
Comment (ahuston-0) [inline systems/configuration.nix:78]: why would you override this???

File: systems/jeeves/configuration.nix [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    firewall.enable = false;
+  };
+
+  boot = {
+    zfs.extraPools = [
+      "Media"
+      "Storage"
+      "Torenting"
+    ];
+    filesystem = "zfs";
```
Comment (ahuston-0) [inline systems/jeeves/configuration.nix:16]: i think this should be ext4 unless you're doing zfs on root. check my config

File: systems/jeeves/configuration.nix [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  virtualisation = {
+    docker = {
+      enable = true;
+      recommendedDefaults = true;
+      logDriver = "local";
+      storageDriver = "overlay2";
+      daemon."settings" = {
+        experimental = true;
+        data-root = "/var/lib/docker";
```
Comment (ahuston-0) [inline systems/jeeves/configuration.nix:28]: this is the default setting. you should be fine to remove this (I only have it so I can point it at a custom directory on ZFS).

File: systems/jeeves/configuration.nix [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    };
+
+    smartd.enable = true;
+
+    sysstat.enable = true;
+
+    usbguard = {
+      enable = false;
+      rules = ''
+        allow id 1532:0241
```
Comment (ahuston-0) [inline systems/jeeves/configuration.nix:82]: !!!!!!!!!

Did you check that this is the correct rule?

pls keep this disabled for now until you check :)

Comment (ahuston-0) [reply systems/jeeves/configuration.nix:82]: granted i see you already have it disabled but still

File: systems/jeeves/configuration.nix [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    zerotierone = {
+      enable = true;
+      joinNetworks = [
+        "e4da7455b2ae64ca"
+        "52b337794f23c1d4"
+      ];
+    };
+  };
+
+  system.stateVersion = "23.11";
```
Comment (ahuston-0) [inline systems/jeeves/configuration.nix:100]: 24.05 is the current unstable. you can probably leave it at 23.11 if you want though

**Review by ahuston-0 — COMMENTED** [review]
boink

File: systems/jeeves/configuration.nix (hunk 2 above)
Comment (ahuston-0) [reply systems/jeeves/configuration.nix:16]: nvm im wrong. resolving

File: systems/configuration.nix (hunk 1 above)
Comment (ahuston-0) [reply systems/configuration.nix:78]: as discussed, pls remove the mkdefault <3

**Review by RichieCahill — APPROVED** [review]

**Review by dkrautha — APPROVED** [review]
Biggest LGTM of my life, send it

**Pull request merged**